src/main/java/sungshin/sooon/controller/AccountController.java

```diff
@@ -18,8 +22,37 @@
 
     // 로그인
     @PostMapping("/login")
-    public ResponseEntity<TokenDto> login(@RequestBody LoginRequest loginRequest) {
-        return ResponseEntity.ok(accountService.login(loginRequest));
+    public ResponseEntity<TokenDto> login(@RequestBody LoginRequestDto loginRequestDto) {
+        return ResponseEntity.ok(accountService.login(loginRequestDto));
+    }
+
+    // 회원가입
+    @PostMapping("/signup")
+    public ResponseEntity signup(@RequestBody @Valid SignUpRequestDto signUpRequestDto) {
+        accountService.signup(signUpRequestDto);
+        return ResponseEntity.ok("회원가입 성공");
+    }
+
+    // 이메일 중복확인
+    @GetMapping("/checkEmail")
```

Comment:
```suggestion
    @GetMapping("/check-email")
```

정확한 의미나 표현을 위해 단어의 결합이 불가피한 경우 반드시 -(dash) 사용한다.

출처: https://sanghaklee.tistory.com/57 [이상학의 개발블로그]

Reply:
수정완료!